src/js/helpers/otherFiltersFormatter.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * otherFiltersFormatter.js
+ * Created by Emily Gullo on 07/10/2017
+ */
+
+import { startCase, toLower } from 'lodash';
+
+export const formatCFDA = (cfda, description) => {
+    let displayValue = `${cfda}`;
+
+    if (description !== null) {
+        displayValue += ` | ${startCase(toLower(description))}`;
+    }
+
+    return displayValue;
+};
+
+export const formatNAICS = (naics, description) => {
+    let displayValue = `${naics}`;
+
+    if (description !== null) {
+        displayValue += ` | ${startCase(toLower(description))}`;
+    }
+
+    return displayValue;
+};
+
+export const formatPSC = (psc) => {
```

Comment:
It's waiting on a secondary value (a description) that is not yet in the DB.  I can use psc directly for the time being and use the new general format function when that arrives.